plugin/evm/config.go

```diff
@@ -264,11 +263,12 @@ func (c *Config) SetDefaults() {
 	c.CommitInterval = defaultCommitInterval
 	c.SnapshotWait = defaultSnapshotWait
 	c.RegossipFrequency.Duration = defaultRegossipFrequency
```

Comment:
May want to move this below the changes to keep ordering from config

Reply:
done